content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation](/tracing/trace_collection/) is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation](/tracing/trace_collection/automatic_instrumentation/) is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries](https://opentelemetry.io/docs/concepts/instrumentation/libraries/) from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
```

Comment:
Currently if a user runs their applications with automatic instrumentation (ex ddtrace-run), the tracer will auto-enable all/most ddtrace integrations. We should provide customer's with steps/examples on how to replace ddtrace integrations with an equivalent opentelemetry instrumentation.

Side note: Most client libraries do not have a single configuration that can disable ALL ddtrace instrumentations. User's must disable ddtrace integrations individually. This user experience seems less than ideal.

Reply:
cc: @tushar-datadog

Reply:
Yes, we should warn about the possibility of duplicate spans here.

But how to resolve that will vary across each language - this is one area of configuration which varies widely from language to language (even in OTel the way you enable/disable Java instrumentations is very specific to the Java tracer.)

Reply:
Added a general warning to the preamble